ray-operator/controllers/ray/common/ingress.go

```diff
@@ -17,8 +17,10 @@ const IngressClassAnnotationKey = "kubernetes.io/ingress.class"
 // This is used to expose dashboard for external traffic.
 func BuildIngressForHeadService(cluster rayiov1alpha1.RayCluster) (*networkingv1.Ingress, error) {
 	labels := map[string]string{
-		RayClusterLabelKey: cluster.Name,
-		RayIDLabelKey:      utils.GenerateIdentifier(cluster.Name, rayiov1alpha1.HeadNode),
+		RayClusterLabelKey:     cluster.Name,
+		RayIDLabelKey:          utils.GenerateIdentifier(cluster.Name, rayiov1alpha1.HeadNode),
+		DefaultAppNameKey:      DefaultApplicationName,
+		DefaultAppComponentKey: DefaultRayClusterControllerName,
```

Comment:
the data plane workloads doesn't need to have `DefaultAppComponentKey`?  you can use `app.kubernetes.io/managed-by` for all resources managed by operators. for the ray cluster, use `app.kubernetes.io/managed-by=kuberay-apiserver`